src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="hmcts-header" role="heading" aria-level="1">
     <div class="hmcts-header__container">
-      <ng-container  *ngIf="isBrandedHeader ;else noBrandedHeader">
+      <ng-container  *ngIf="isBrandedHeader && headerTitle.hideBranding !== true;else noBrandedHeader">
```

Comment:
`*ngIf="isBrandedHeader && !headerTitle.hideBranding; else noBrandedHeader"` this can be simpified

Reply:
Ah, I suggested doing it with `!== true`, @kuda1992. I thought it'd just make it more explicit but either way works fine. :)